backend/src/tracking/tracking/tracking.service.ts

```diff
@@ -667,7 +667,7 @@ export class ParcelLabTrackingService {
       recipient: this.getName(shopifyOrder),
       recipient_notification: this.getName(shopifyOrder),
       statuslink: shopifyOrder?.order_status_url,
-      street: shopifyOrder?.shipping_address?.address1,
+      street: `${shopifyOrder?.shipping_address?.address1}\n${shopifyOrder?.shipping_address?.address2}`,
```

Comment:
@trollkotze can you check if shopifyOrder.shipping_address.address2 is set and not falsy / empty?

Otherwise your changes will always add a new line, also if there is no address2.

Reply:
Okay.

Reply:
Same on line 585